pkgs/by-name/mu/muvm/package.nix

```diff
@@ -9,44 +9,44 @@
   passt,
   sommelier,
   mesa,
+  systemd,
+  udev,
+  pkg-config,
   opengl-driver ? mesa.drivers,
   withSommelier ? false,
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "krun";
-  version = "0-unstable-2024-06-18";
+  pname = "muvm";
+  version = "0.2.0";
 
   src = fetchFromGitHub {
-    owner = "slp";
+    owner = "AsahiLinux";
     repo = pname;
```

Comment:
```suggestion
    repo = "muvm";
```

Reply:
Is this change really necessary? `pname` is already `muvm`.

Reply:
Purely cosmetic now, but can be problematic in combination with finalAttrs in the future: https://github.com/NixOS/nixpkgs/issues/310373

Reply:
Ok but we're using `buildRustPackage` which doesn't support `finalAttrs`, is there plans to make `buildRustPackage` support `finalAttrs` or has that already happened?

Reply:
It SHOULD happen I guess? Not aware of the progress though.